lambdas/call-ticf-cri/src/main/java/uk/gov/di/ipv/core/callticfcri/service/TicfCriService.java

```diff
@@ -73,9 +73,10 @@ public List<VerifiableCredential> getTicfVc(
             ClientOAuthSessionItem clientOAuthSessionItem, IpvSessionItem ipvSessionItem)
             throws TicfCriServiceException {
         try {
-            RestCriConfig ticfCriConfig = configService.getRestCriConfig(TICF);
+            var connection = configService.getActiveConnection(TICF);
```

Comment:
This could actually have been a (very rare!) bug - we could have ended up using a different connection for the main configuration vs. the API key